sdk/tables/azure-data-tables/tests/test_table_client_cosmos.py

```diff
@@ -175,6 +175,28 @@ def test_table_name_errors_bad_length(self, tables_cosmos_account_name, tables_p
             client.submit_transaction(batch)
         assert error.value.error_code == "ResourceNotFound"
 
+    @cosmos_decorator
+    @recorded_by_proxy
+    def test_table_client_location_mode(self, tables_cosmos_account_name, tables_primary_cosmos_account_key):
+        url = self.account_url(tables_cosmos_account_name, "cosmos")
+        table_name = self.get_resource_name("mytable")
+        entity = {"PartitionKey": "foo", "RowKey": "bar"}
+
+        with TableClient(
+            url, table_name, credential=tables_primary_cosmos_account_key, location_mode=LocationMode.SECONDARY
+        ) as client:
+            client.create_table()
```

Comment:
I would like to see the recordings for these - I'm curious if we're actually using the "-secondary" endpoint or not....

Reply:
Just uploaded recordings, it is using "-secondary" endpoint: https://github.com/Azure/azure-sdk-assets/blob/python/tables/azure-data-tables_4772ff04b7/python/sdk/tables/azure-data-tables/tests/recordings/test_table_client_async.pyTestTableClientAsynctest_table_client_location_mode.json